sections/main-collection.liquid

```diff
@@ -45,5 +45,3 @@
   ]
 }
 {% endschema %}
-
-{{ 'queries-handler.js' | asset_url | script_tag_deferred }}
```

Comment:
why did we remove this

Reply:
if you check in our main u will see that` {{ 'queries-handler.js' | asset_url | script_tag_deferred }}` included two times so that the reason we got an error in the console say that some variables already exist

Reply:
Specifically inside : `main-pagination.liquid `